src/oc_bifrost/oc-bifrost-pedant/spec/bifrost/groups_spec.rb

```diff
@@ -891,4 +891,95 @@ def self.creates_group_as(requestor, headers = {})
       end # for groups
     end # DELETE
   end # /groups/<group_id>/<member_type>/<member_id>
+
+  context '/groups/<group_id>/transitive_member/<member_type>/<member_id>' do
+    should_not_allow :PUT,
+      "/groups/eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee/transitive_member/actors/ffffffffffffffffffffffffffffffff"
```

Comment:
oh i don't mean refactor it, just add a note about why it is that long name? It was just confusing to me.